CMakeLists.txt

```diff
@@ -614,26 +636,21 @@ if(OPENAL_FOUND)
 	endif()
 endif()
 
-if(NOT ANDROID)
-	if(NOT APPLE)
-		include(FindSDL2 OPTIONAL)
-	endif()
+if(ENABLE_SDL OR (NOT APPLE AND NOT ANDROID AND NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Linux" AND NOT MSVC))
```

Comment:
Is it possible to move this plattform code to line 16, so it would be possible to disable SDL on freebsd.